### PR TITLE
webhooks: Move init code into webhooks package

### PR DESCRIPTION
**Pull request by ryanslade**
The init code in the `repos` package was webhooks specific, so move it into the `webhooks` package.

## Test plan

Mechanical move, all tests pass

### CODE REVIEW

File: enterprise/cmd/frontend/shared/shared.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ var initFunctions = map[string]EnterpriseInitializer{
 	"licensing":      licensing.Init,
 	"notebooks":      notebooks.Init,
 	"searchcontexts": searchcontexts.Init,
-	"repos":          repos.Init,
+	"repos.webhooks": webhooks.Init,
```
Comment (sashaostrikov) [inline enterprise/cmd/frontend/shared/shared.go:54]: Should it be "webhooks" instead of "repos.webhooks"?

**Review by sashaostrikov — APPROVED** [review]
:shipit:

File: enterprise/cmd/frontend/shared/shared.go (hunk 1 above)
Comment (ryanslade) [reply enterprise/cmd/frontend/shared/shared.go:54]: It's not the only place we do webhooks stuff, we also do that for batch changes and some authz stuff. In this case the init code is specifically for repo related webhooks.

Comment (sashaostrikov) [reply enterprise/cmd/frontend/shared/shared.go:54]: :shipit::shipit::shipit:

**Pull request merged**